src/main/steps/applicant1/connection-summary/content.ts

```diff
@@ -3,36 +3,35 @@ import { TranslationFn } from '../../../app/controller/GetController';
 import { FormContent } from '../../../app/form/Form';
 import { CommonContent } from '../../common/common.content';
 
-export const jurisdictionMoreDetailsContent: (formState) => { connectedToEnglandWales: string; readMore: string } =
-  formState => {
-    const resConnection = enContainsHabitualResConnection(formState.connections);
-    const domConnection = enContainsDomConnection(formState.connections);
+export const jurisdictionMoreDetailsContent: (
+  formState,
+  isRespondent
+) => { connectedToEnglandWales: string; readMore: string } = (formState, isRespondent) => {
```

Comment:
i think isRespondent should `have` a default value
```suggestion
) => { connectedToEnglandWales: string; readMore: string } = (formState, isRespondent = false) => {
```